Application/Server/src/ClientRequestManager.cpp

```diff
@@ -0,0 +1,177 @@
+#include <sstream>
+
+#include "ClientRequestManager.h"
+#include "UserDBManager.h"
+#include "MenuDBManager.h"
+#include "UserDTO.h"
+#include "RolloutDBManager.h"
+#include "VoteDBManager.h"
+#include "FeedbackDBManager.h"
+#include "RecommendationEngine.h"
+#include "NotificationDBManager.h"
+
+std::vector<std::string> splitStringbyTab(const std::string& str) {
+    std::vector<std::string> result;
+    std::stringstream ss(str);
+    std::string token;
+    
+    while (std::getline(ss, token, '\t')) {
+        result.push_back(token);
+    }
+    
+    return result;
+}
+
+std::string ClientRequestManager::loginRequest(std::string message) {

```

Comment:
Again, the method name here is a noun which should instead be a verb. Please update everywhere.